File: P02 - Kelas Objek/Python/Bag. pertama/Kelas03.py
# File: Kelas03.py
# Penulis : Elmosius Suli
# Docstrings untuk menyisipkan dokumentasi langsung di dalam program

class Kelas03:
    """
    Kelas03 digunakan untuk menunjukkan docstrings.
    Jika docstrings lebih dari 1 baris, 
    maka bisa menggunakan 3 buah tanda kutip ganda
    atau tanda kutip tunggal, seperti ini.
    """
    pass
# Ini sudah diLuar KeLasO3

## Program utama ##
def main():
    "Ini docstrings di dalam main()"
    k = Kelas03()
    
# Perintah untuk menjaLankan method main() hanya jika keLas di run sendiri,
# bukan di import dari keLas Lain.

if __name__ == '__main__':
    main()
    help(Kelas03) # MenampiLkan docstrings
    help(main)

File: P02 - Kelas Objek/Java/Bag. pertama/Kelas03.java
// Menyisipkan dokumentasi langsung di dalam program & di generate dg Javadoc

/**
 * Ini adalah bentuk komentar di Java yg dapat di generate
 * langsung menjadi dokumentasi dengan menggunakan program javadoc.
 * Formatnya /** .... * /
 */

 /** 
  * <b> Kelas03 </b> adalah kelas untuk menunjukkan javadoc <br>
  * Dapat menggunakan tag2 html <br>
  * Ada annotasi2 khusus (tanda @) yg memiliki arti tertentu <br>
  * Dari cmd eksekusi dengan: javadoc Kelas03.java <br>
  * Dari IntelliJ menu: Tools | Generate Javadoc... <br>
  *
  * @author Elmo  
  * @version 1.0
  * @since 27/09/2023 
 */
package Praktikum.P02;

public class Kelas03 {
    /**
     * Ini adalah method main() <br>
     * @param args Nothing 
     */
    public static void main(String[] args){
        System.out.println("Hallo Dunia... mari belajar Java!");
    }
}
